package-lock.json

```diff
@@ -5135,13 +5133,12 @@
       }
     },
     "gl-surface3d": {
-      "version": "1.4.5",
-      "resolved": "https://registry.npmjs.org/gl-surface3d/-/gl-surface3d-1.4.5.tgz",
-      "integrity": "sha512-uu+s3jxM7kMdTpYvXVJPujOQMo3B/+AOG4KX7niZHVnykuCKO11+haZSDF2ucymrIrRyUGknGIto6CVpLmP7og==",
+      "version": "git://github.com/gl-vis/gl-surface3d.git#b82222510dcb6a0bc741db0da4e009352ff97242",
+      "from": "git://github.com/gl-vis/gl-surface3d.git#b82222510dcb6a0bc741db0da4e009352ff97242",
       "requires": {
         "binary-search-bounds": "^2.0.4",
         "bit-twiddle": "^1.0.2",
-        "colormap": "^2.1.0",
+        "colormap": "git://github.com/archmoj/colormap.git#c8b31f77f88ba1160f30bcd6ba10753ca0ed0404",
```

Comment:
https://github.com/bpostlethwaite/colormap/compare/master...archmoj:master

Reply:
:ok_hand: can you make a PR there and ping @bpostlethwaite ?